fvm/fvm_fuzz_test.go

```diff
@@ -147,9 +152,36 @@ var fuzzTransactionTypes = []transactionType{
 			codes := []errors.ErrorCode{
 				errors.ErrCodeComputationLimitExceededError,
 				errors.ErrCodeCadenceRunTimeError, // because of the panic
+				errors.ErrCodeLedgerInteractionLimitExceededError,
```

Comment:
This fuzzy test says if there is an error, then the error must be one of them. 

However, if interaction limit exceeded error was hit where it shouldn't, this fuzzy test would still pass. 

I wonder do we have another non-fuzzy test case that verifies if the interaction limit is exceeded, it will report interaction limit exceeded error?

Reply:
yes, there are regular tests for this https://github.com/onflow/flow-go/blob/82493852fa223726a11a3beca11e44f6fc1105af/fvm/fvm_blockcontext_test.go#L999. 

We can improve the tests later to also predict which error is expected with which parameters.